### DISCUSSION TITLE
How to get 2 notes to play as a chord type

### DISCUSSION THREAD

**Discussion by ReneDiazDiversity** [tonaljs/tonal#300] [Q&A]
\I'm having trouble with Tonal JS playing two notes as a chord. e.g a perfect fifth aka a power chord or Major and Minor 3rds as double stops (aka 2 note chords)

How do we get play back of two-note intervals as chords?

Currently, we can use Chord.getChord('major', 'C') to get the needed notes info to play a chord. But when we run Chord.getChord('3M', 'C') or Chord.getChord('6m', 'C') we get back and empty chord.

Is there a Tonal JS package or function we are missing to get note information with an input argument like '3M', 'C' or '6m', 'C' to return an interval for example 3M for a Major third chordal interval, and to return an interval for example 6m for a minor 6 chordal interval?

**Comment by danigb** [comment]
Hi, chords uses standard chord names, so `getChord('major', 'C')` is the same as `getChord('M', 'C')`. For minor sixth chords you may write `getChord('m6', 'C')` or `getChord('Cm6')`

But maybe instead of chords, you're looking for interval transposition: `Note.transpose('C', '3M')`

Hope it helps
